android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/UnityMessage.java

```diff
@@ -0,0 +1,22 @@
+package com.shopify.unity.buy;
+
+import com.google.gson.Gson;
+import java.util.UUID;
+
+public class UnityMessage implements JsonSerializable {
+
+    // Using capital case here, so when it is serialized the keys are capitalized
+    public String Content;
```

Comment:
Related to the checkstyle errors, we can probably make these lowercased and set the JSON object keys in the `toJsonString` function to be captialized - similar to how we've done in Unity.